src/thunderbots/software/util/parameter/dynamic_parameters.h

```diff
@@ -0,0 +1,199 @@
+
+/**
+ *  !! WARNING !!
+ *
+ *  THIS FILE IS AUTOGENERATED, ANY CHANGES MADE WILL BE LOST
+ *
+ *  !! WARNING !!
+ */
```

Comment:
Similar comment to above, we should remove this autogenerated file

Reply:
Removed the autogenerated file